cli-decoder/README.md

```diff
@@ -0,0 +1,24 @@
+
+## Test task:
+Develop a cli tool that will:
+
+- that parses cmd line tools. It should be either `json` or `xml`. 
+- if JSON is selected - cmd line tool should only accept valid json string, same goes to xml.
+- After data is received and validated it should be stored as file in current dir with json or xml format.
+- Data is saved only if it's unique. 
+- Uniquness should be identified using md5 has of a file.
+- On startup identify all known md5 hashsums
+- Put hash data into the separate file and keep them in memory when program is running  
+- If error appeared - program should still be working and accept new data. Handle all the errors and panics properly
+
+## Preparation steps
+0. Initialize go mod
+1. Install https://github.com/spf13/cobra and read the documentation
```

Comment:
is it important which `cli` lib use? We may let to choose. https://github.com/urfave/cli for example also has a good rating and fewer dependencies.

Reply:
ye, agree, not really important.
I decided to put cobra as it's more popular  :)